.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,13 @@
+---
+name: ✨ Feature request
+about: Suggest an idea for this project
+labels: feature request
+
+---
+
+## Summary
+<!-- include a summary -->
```

Comment:
Maybe ask for the api to wrap or add a disclamer that we will not collect pulpcore feature requests here.